rpcs3/Emu/Cell/lv2/sys_usbd.cpp

```diff
@@ -175,7 +175,8 @@ usb_handler_thread::usb_handler_thread()
 
 		check_device(0x0E6F, 0x0241, 0x0241, "Lego Dimensions Portal");
 		check_device(0x0E6F, 0x0129, 0x0129, "Disney Infinity Portal");
-
+		check_device(0x0E6F, 0x200A, 0x200A, "Kamen Rider Summonride Portal");
+		
```

Comment:
Why is there an extra line change? Are you using different line endings?

Reply:
It is my fault I'm editing from a smartphone. If this need a change let me know.